identity/webapp/pages/error.tsx

```diff
@@ -22,16 +27,23 @@ const ErrorPage: NextPage<Props> = ({ errorDescription }) => {
             <Wrapper>
               <SectionHeading as="h1">An error occurred</SectionHeading>
 
-              <p>{errorDescription}</p>
-              <OutlinedButton>
-                <a
-                  href="mailto:library@wellcomecollection.org"
-                  target="_blank"
-                  rel="noopener noreferrer"
-                >
-                  Contact us
-                </a>
-              </OutlinedButton>
+              {hasCustomErrorDescription ? (
+                <p dangerouslySetInnerHTML={{ __html: errorDescription }} />
+              ) : (
```

Comment:
This makes me a bit uncomfortable, as `errorDescription` _can_ come from the user, and a bug in how we set `hasCustomErrorDescription` - the initialisation of which is a bit divorced from its use - would have quite bad consequences here.

One tempting option is to avoid `dangerouslySetInnerHTML` by just having the email as a string (as we've done elsewhere), another would be to forego the ease of editing make the custom error message a React component with a normal `<a>`...

Reply:
Yeah, this also set minor alarm bells ringing for me and prompted my wondering about future pen testing.